workloads/templates/vm-creation.md

```diff
@@ -17,53 +17,79 @@ Here is an example template that defines an instance of the `VirtualMachine` obj
 apiVersion: v1
 kind: Template
 metadata:
-  name: fedora-vmi-template
   annotations:
-    description: "OpenShift KubeVirt Fedora VM template"
-    tags: "kubevirt,openshift,template,linux"
+    description: OCP KubeVirt Fedora 27 VM template
+    iconClass: icon-fedora
+    tags: kubevirt,ocp,template,linux,virtualmachine
+  creationTimestamp: null
```

Comment:
```suggestion
```

Reply:
There is really no need to add null elements to the template. Or is there?

Reply:
In this case also good idea to send MR to change files:

```bash
workloads/controllers/virtual-machine.md:123:  creationTimestamp: null
workloads/controllers/virtual-machine.md:131:      creationTimestamp: null
workloads/virtual-machines/unresponsive-nodes.md:22:    creationTimestamp: 2018-11-05T08:55:53Z
workloads/virtual-machines/image-upload.md:88:  creationTimestamp: null
workloads/virtual-machines/disks-and-volumes.md:399:  creationTimestamp: null
workloads/virtual-machines/disks-and-volumes.md:514:  creationTimestamp: null
workloads/virtual-machines/disks-and-volumes.md:594:  creationTimestamp: null
workloads/virtual-machines/disks-and-volumes.md:655:  creationTimestamp: null
workloads/virtual-machines/disks-and-volumes.md:780:  creationTimestamp: null
workloads/virtual-machines/disks-and-volumes.md:868:  creationTimestamp: null
workloads/virtual-machines/disks-and-volumes.md:1030:  creationTimestamp: null
```

Reply:
@MarSik ?

Reply:
Sure why not, but it is not a part of this PR, just do not introduce any more..